lib/sessions.js

```diff
@@ -17,9 +19,10 @@ const sessions = new Map();
 
 const generateToken = () => {
   const base = ALPHA_DIGIT.length;
+  const bytes = crypto.randomBytes(base);
   let key = '';
   for (let i = 0; i < TOKEN_LENGTH; i++) {
-    const index = Math.floor(Math.random() * base);
+    const index = ((bytes[i] * base) / BYTE) | 0;
```

Comment:
I am ok with it, but `x | 0` truncates `x`, it's asm.js equivalent for `Math.truncate`. I think in this case it is not important. Also we have `~~x` and it's also suitable here but not readable.

Reply:
I was guided by the fact that "bitwise or" seems to be the fastest way (~2% than Math(floor or trunc), by sometimes jsperf tests x | 0 fastest than Math.floor or Math.trunk at 30%)

Reply:
It's ok here but in this starter kit I am trying to write readable code, not performance-optimized